fdbserver/DDTeamCollection.h

```diff
@@ -595,6 +595,12 @@ class DDTeamCollection : public ReferenceCounted<DDTeamCollection> {
 
 	void addTeam(std::set<UID> const& team, bool isInitialTeam) { addTeam(team.begin(), team.end(), isInitialTeam); }
 
+	// FIXME: Public for testing only
+	void enableBuildingTeams(bool doBuildTeams) { this->doBuildTeams = doBuildTeams; }
+
+	// FIXME: Public for testing only
+	void clearCheckTeamDelay() { this->checkTeamDelay = Void(); }
```

Comment:
This should be called `setCheckTeamDelay()`, because the future is being set. Clearing would be:
```
this->checkTeamDelay = Future<Void>();
```